src/routes/userRouter.ts

```diff
@@ -0,0 +1,10 @@
+import { Router } from "express";
+import userControllers from "../modules/user/controller/userControllers";
+import {isUserExist} from "../middlewares/validation";
+import { validateUserStatus } from "../modules/user/validation/userValidations";
+
+const router: Router = Router()
+
+router.put("/admin-update-user-status/:id", validateUserStatus, isUserExist, userControllers.updateUserStatus);
```

Comment:
Please use Joi Validation as we used into Register, Login, Send Verification Email, etc..